text/3-entrees/storage-api/ringbuffer.md

```diff
@@ -188,10 +241,15 @@ where // ... same where clause elided
 }
 ```
 
-On `drop`, we `commit` the bounds to storage. With this implementation of `Drop`, `commit` is called when our transient goes out of scope, making sure that the storage state is consistent for the next call to the using pallet.
+On `drop`, we `commit` the bounds to storage. With this implementation of `Drop`, `commit` is called
+when our transient goes out of scope, making sure that the storage state is consistent for the next
+call to the using pallet.
```

Comment:
There are some situations where `drop()` is not guaranteed to run but not sure if that is something we care about mentioning here (OOM, panics in the the `drop()` impl or calls to `mem::forget()`).

Reply:
Runtimes are already assumed not to OOM or painc.

Not sure about mem::forget(), but I think it's safe to to say if you're using mem::forget you ought to know the risks.